apps/web/src/server/applications/create/check-your-answers/__tests__/applications-create-check-your-answers.test.js

```diff
@@ -0,0 +1,84 @@
+import { parseHtml } from '@pins/platform';
+import nock from 'nock';
+import supertest from 'supertest';
+import { createTestApplication } from '../../../../../../testing/index.js';
+
+const { app, installMockApi, teardown } = createTestApplication();
+const request = supertest(app);
+const successGetResponse = { id: 1, applicants: [{ id: 1 }] };
+const successPostResponse = { id: 1, reference: 'AB0110203', status: 'Pre-Application' };
+const errorPostResponse = {
+	errors: {
+		projectLocation: 'Missing projectLocation',
+		subSector: 'Missing subSector',
+		sector: 'Missing sector',
+		regions: 'Missing regions',
+		gridReferenceEasting: 'Missing gridReferenceEasting',
+		gridReferenceNorthing: 'Missing gridReferenceNorthing'
+	}
+};
+
+const nocks = () => {
+	nock('http://test/').get('/applications/case-officer').reply(200, successGetResponse);
+	nock('http://test/').get('/applications/123').times(2).reply(200, successGetResponse);
```

Comment:
I don't believe that the `successGetResponse` object matches the structure of the data that we would receive from the API?

Reply:
we removed it